src/node_nanomsg.cc

```diff
@@ -23,6 +23,11 @@ using v8::Object;
 using v8::String;
 using v8::Value;
 
+#define ret NanReturnValue
+#define utf8 v8::String::Utf8Value
```

Comment:
> haha yup good call, btw what do you think about that #define S

I see that you mimic what's going on in the man page.  I don't know how I feel about 1 character macros, but it's fine.

> personally, i was using As<Number>()->IntegerValue() which may do a 64 bit conversion on systems where int can be 64... i think it falls back to 32 otherwise

I would think that `NanNew<Number>()` should handle the conversions correctly.